aspnetcore/signalr/hubs.md

```diff
@@ -37,9 +37,29 @@ Create a hub by declaring a class that inherits from `Hub`, and add public metho
 
 You can specify a return type and parameters, including complex types and arrays, as you would in any C# method. SignalR handles the serialization and deserialization of complex objects and arrays in your parameters and return values.
 
+## The Context object
+
+Each instance of the `Hub` class has a property named `Context` that contains the following properties with information about the connection:
```

Comment:
I prefer
Each instance of the `Hub` class has a `Context` property with the following properties which contain information about the connection:

I wonder if we even need each instance.

`Hub.Context` has the following properties which contain information about the connection:

or ~which~ containing information ..